bin/migration/tables/roles.py

```diff
@@ -0,0 +1,41 @@
+from .helpers import check_existing_record, audit_entry_creation, log_failed_imports
+import uuid
+
+class RoleManager:
+    def __init__(self):
+        self.failed_imports = set()
+
+    def migrate_data(self,destination_cursor):
+        allowed_roles = ['Level 1', 'Level 2', 'Level 3', 'Level 4', 'Super User']
+        batch_roles_data = []
+
+        for role in allowed_roles:
+            id = str(uuid.uuid4())
+            name = role
+
+            if name in allowed_roles and not check_existing_record(destination_cursor,'roles', 'id', id):
+                batch_roles_data.append((id, name))
+
+        try:
+            if batch_roles_data:
+                destination_cursor.executemany(
```

Comment:
@mazzopardi2 Customised 'Security' roles except ‘Level1/2/3/4 and superuser’ are not migrated .

Reply:
@mazzopardi2 grouptype and groupdescription values not being included at the migration. Are we going to migrate them ?

Reply:
With recent commits, we are now pulling the Security roles from the grouplist table.
I have also included a description field in the roles table to include the role descriptions as discussed with Lawrie.
Thank you